test/utils/DateFormatterSpec.scala

```diff
@@ -16,37 +16,49 @@
 
 package utils
 
-import java.time.format.DateTimeFormatter
-import java.time.{LocalDate, LocalDateTime}
+import java.time.LocalDate
+
+import play.api.i18n.{Lang, Messages}
 
 class DateFormatterSpec extends TestSupport with ImplicitDateFormatter {
 
-  "The implicit date formatter" should {
+  private trait Test {
+    implicit lazy val msgs: Messages = Messages(new Lang("en"), messagesApi)
+  }
 
-    "format string dates" in {
-      "2017-04-01".toLocalDate shouldBe LocalDate.of(2017, 4, 1)
-    }
+  "The implicit date formatter" when {
 
-    "format months with single digit values" in {
-      "2017-6-30".toLocalDate shouldBe LocalDate.of(2017, 6, 30)
-    }
+    "given a date as a string" should {
 
-    "format days with single digit values" in {
-      "2017-6-1".toLocalDate shouldBe LocalDate.of(2017, 6, 1)
-    }
+      "convert it to a LocalDate" in new Test{
+        "2017-04-01".toLocalDate shouldBe LocalDate.of(2017, 4, 1)
```

Comment:
Could you change the LocalDate.of to LocalDate.parse just to avoid the magic number warnings